Makefile

```diff
@@ -17,6 +17,10 @@ PROJECT := kilo
 PKG := github.com/squat/$(PROJECT)
 REGISTRY ?= index.docker.io
 IMAGE ?= squat/$(PROJECT)
+ifneq ($(REGISTRY),index.docker.io)
```

Comment:
Would you mind splitting this up into a new PR so we can keep each chunk of work separate and still squash the commits related to NAT?

Reply:
done #147

Reply:
I missed the drop from this PR, sorry...